src/client/containers/ShoppingCartPage/ShoppingCartPage.styles.css

```diff
@@ -1,3 +1,21 @@
 div {
   color: #687808;
 }
+
+.shop-cart {
+  margin: 50px;
+  padding: 0;
+  width: 40%;
+}
+.heading {
+  text-align: center;
```

Comment:
.heading can be inconsistent in some instances. Please give very specific classNames.